docs/examples/tutorial/pure/cast.py

```diff
@@ -0,0 +1,4 @@
+import cython
+obj = 123
+i1 = cython.cast(cython.int, obj)
+i2 = cython.cast(cython.int, obj, typecheck=True)
```

Comment:
```suggestion
obj = [1, 2, 3]
l1 = cython.cast(list, obj)
l2 = cython.cast(list, obj, typecheck=True)
```

I'll propose changing this to a list because the check here actually makes a difference - it doesn't seem to for `int`